2016-17/C01 Make Episode File.sps

```diff
@@ -354,11 +364,11 @@ aggregate
     /Break chi cij_marker
     /cij_ppa = Max(PPA).
 
-sort cases by chi CIJ_start_date.
+sort cases by chi keydate1_dateformat.
 
 *  We are not currently including Social Care data for some years but we still want the variables for consistency.
 * Social Care.
- * String
+ String
```

Comment:
Same as 1516 the CH and sc_send_lca should already exists

Reply:
CH variables added back in 1617 file, not too sure about numeric variables. Can you please double check what i have added back in/deleted is correct